solc/CommandLineInterface.cpp

```diff
@@ -872,9 +873,9 @@ General Information)").c_str(),
 		)
 		(
 			g_strEVMVersion.c_str(),
-			po::value<string>()->value_name("version"),
+			po::value<string>()->value_name("version")->default_value(EVMVersion::berlin().name()),
```

Comment:
So it turns out CompilerStack/StandardCompiler just uses `EVMVersion()` (the default constructor) to get the default. Probably should do the same here to avoid inconsistency.

Reply:
I don't mind either. But @cameel suggested that this is better since we don't need to write "default" next to the version in the description.

Reply:
Well, `EVMVersion().name()` would work too. Maybe I should have suggested it in the first place. It does save us the hassle of updating it if the default changes (we'll still have to update the description if we add a new EVM version though).

Reply:
Changed it to `EVMVersion().name()`.